db/migrate/20250610095041_add_flexible_page_content_to_editions.rb

```diff
@@ -0,0 +1,9 @@
+class AddFlexiblePageContentToEditions < ActiveRecord::Migration[8.0]
+  def change
+    change_table :editions, bulk: true do |t|
```

Comment:
Did we consider using the existing `type` property for the flexible page type, and reusing a `body` property for storing the JSON? I think that was the original idea (for POC at least) and it would be good to expand on alternatives considered in the commit message.

Aside: not sure which 'body' property we'd have hooked into in the end - the one in govspeak_contents or the one in edition_translations? 🤔